pkg/knuu/knuu.go

```diff
@@ -54,10 +56,72 @@ func InitializeWithIdentifier(uniqueIdentifier string) error {
 		return err
 	}
 
+	// read timeout from env
+	timeoutString := os.Getenv("KNUU_TIMEOUT")
+
+	if timeoutString == "" {
+		timeout = 60 * time.Minute
+	} else {
+		parsedTimeout, err := time.ParseDuration(timeoutString)
+		if err != nil {
+			return fmt.Errorf("cannot parse timeout: %s", err)
+		}
+		timeout = parsedTimeout
+	}
+
+	if err := handleTimeout(); err != nil {
+		return fmt.Errorf("cannot handle timeout: %s", err)
+	}
+
 	return nil
 }
 
 // IsInitialized returns true if knuu is initialized, and false otherwise
 func IsInitialized() bool {
 	return k8s.IsInitialized()
 }
+
+// handleTimeout creates a timeout handler that will delete all resources with the identifier after the timeout
+func handleTimeout() error {
+
+	instance, err := NewInstance("timeout-handler")
+	if err != nil {
+		return fmt.Errorf("cannot create instance: %s", err)
+	}
+	// FIXME: use supported kubernetes version images (use of latest could break)
```

Comment:
make a todo with issue in the repo and link it here
We will remember to do this nit later on

Reply:
linked the issue, thanks for that :+1: